kernel-python/declarativewidgets/urth_widget.py

```diff
@@ -1,14 +1,15 @@
 # Copyright (c) Jupyter Development Team.
 # Distributed under the terms of the Modified BSD License.
 import time
+import logging
 
 from ipywidgets import widgets  # Widget definitions
 
-
 class UrthWidget(widgets.Widget):
     """ A base class for Urth widgets. """
 
     def __init__(self, **kwargs):
+        self.log.setLevel(logging.ERROR)
```

Comment:
Hmm... is it ok to hardcode the log level here? Shouldn't these be something configurable from the outside?

/cc @parente Thoughts?